src/test/java/com/mongodb/kafka/connect/sink/writemodel/strategy/CustomDeleteWriteModelStrategy.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.kafka.connect.sink.writemodel.strategy;
+
+import org.bson.BsonDocument;
+
+import com.mongodb.client.model.WriteModel;
+
+import com.mongodb.kafka.connect.sink.converter.SinkDocument;
+
+public final class CustomDeleteWriteModelStrategy implements WriteModelStrategy {
```

Comment:
Is this a new write model strategy? Not able to find it in the docs - https://www.mongodb.com/docs/kafka-connector/current/sink-connector/configuration-properties/write-strategies/#strategies
In case it is, it needs to be updated in mongo-kafka-private as well

Reply:
No, it is not a new write model strategy, it is for the tests added in this PR -> https://github.com/confluentinc/mongo-kafka/pull/44/files#diff-036219cb0df9f3a2f49bee05e223149d0d40921953c4582e63141ee5b3fa89f3R861